src/styles/block/clock.scss

```
@@ -0,0 +1,43 @@
+@import "../utils/mixin";
+@import "../utils/variables";
+
+.clock {
+
+  display: flex;
+  margin: 0 auto;
+  background-color: #e5e4e2;
+  @include circle($clock-size);
+  justify-content: center;
+  align-items: center;
+  border: 4px solid #98afc7;
```

Also check your properties order https://9elements.com/css-rule-order/

```suggestion
  display: flex;
  margin: 0 auto;
  background-color: #e5e4e2;
  @include circle($clock-size);
  justify-content: center;
  align-items: center;
  border: 4px solid #98afc7;
```